_sass/profile.scss

```diff
@@ -1,5 +1,5 @@
 .profile {
-    background-color: #133667 !important;
+    background-color: #133667 ;
```

Comment:
I think I missed `!important` here.

Reply:
Will add this in next commit.

Reply:
Are you pushing this change or should I?